src/silly_sprites/application.hpp

```diff
@@ -35,8 +37,11 @@ namespace sly {
     private:
         ApplicationSettings m_settings;
         StopWatch m_stopwatch;
-        std::vector<std::unique_ptr<Scene>> m_scenes;
+        gl::Window m_window;
+        gl::Renderer m_renderer;
+        std::unique_ptr<gl::ShaderProgram> m_shader_program;
         std::unique_ptr<script::Engine> m_script_engine;
+        std::vector<std::unique_ptr<Scene>> m_scenes;
 
     public:
         explicit Application(ApplicationSettings settings);
```

Comment:
should we delete copy and move ctor and copy and move assign here?
it makes so sens to copy or move an application right?

Reply:
Good idea, done.